src/lib/FlightTasks/tasks/ManualStabilized/FlightTaskManualStabilized.cpp

```diff
@@ -123,9 +123,10 @@ void FlightTaskManualStabilized::_updateThrustSetpoints()
 
 	/* The final thrust setpoint is found by rotating the scaled unit vector pointing
 	 * upward by the Axis-Angle.
+	 * Make sure that the attitude can be controlled even at 0 throttle.
 	 */
 	Quatf q_sp = AxisAnglef(v(0), v(1), 0.0f);
-	_thrust_setpoint = q_sp.conjugate(Vector3f(0.0f, 0.0f, -1.0f)) * _throttle;
+	_thrust_setpoint = q_sp.conjugate(Vector3f(0.0f, 0.0f, -1.0f)) * math::max(_throttle, 0.0001f);
```

Comment:
With the current FlightTask structure, I think that would be the easiest fix. If you think that the flight performance is different now, then we probably need to add a special case when the parameter is set to 0

Reply:
> If you think that the flight performance is different now, then we probably need to add a special case when the parameter is set to 0

What do you mean by different? With this fix it flies as expected. I was just wondering if it's a more general problem that needs to be fixed in other FlightTasks as well.

Reply:
I think we are talking at cross purposes.
There are two things i wanted to say: 
1. I don't have a better solution as the one you have provided. The only other option would be to create a special case, but I prefer your solution as long as it works.
2. other flighttasks might be effected: please look at https://github.com/PX4/Firmware/pull/10545#issuecomment-423890137

Reply:
Ah sorry I didn't see your edit. All good then.